src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -69,7 +69,7 @@
     // zip file which contains all the report template files
     private static final String TEMPLATE_FILE = "/templateZip.zip";
     private static final String INDEX_PAGE_TEMPLATE = "index.html";
-    private static final String INDEX_PAGE_DEFAULT_TITLE = "<title>RepoSense Report</title>";
+    private static final String INDEX_PAGE_DEFAULT_TITLE = "<title>reposense</title>";
```

Comment:
I feel like I should explain this. The default title in the template index.js reflects the project name in package.json which currently is `reposense`. So, it should replace `"<title>reposense</title>"` and not `"<title>RepoSense Report</title>"` with the report config title. The report config title defaults to `RepoSense Report` so the the title in index.html after replacement will be `RepoSense Report`

Reply:
Can I confirm if the replacement is done in line 180 of this file?

Reply:
Yes, line 180 is the only reference to `INDEX_PAGE_DEFAULT_TITLE`. The replacement is done along with `prepareTemplateFile`.